package.json

```diff
@@ -166,11 +169,11 @@
     "rewire": "2.5.2",
     "rimraf": "2.6.2",
     "sass-lint": "1.12.1",
-    "sass-loader": "6.0.6",
+    "sass-loader": "6.0.7",
     "sinon": "4.1.4",
     "source-map-loader": "0.2.3",
     "start-server-and-test": "1.10.0",
-    "style-loader": "0.19.1",
+    "style-loader": "2.0.0",
```

Comment:
Didn't we remove style loader in web pack config?

Reply:
That's right! Gone